tests/k8st/utils/utils.py

```diff
@@ -198,18 +223,28 @@ def generate_unique_id(length, prefix=""):
     random_string = ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(length))
     return "%s-%s" % (prefix, random_string)
 
+ipv6_map = {
+    "kind-control-plane": "2001:20::8",
+    "kind-worker": "2001:20::1",
+    "kind-worker2": "2001:20::2",
+    "kind-worker3": "2001:20::3",
+}
+
```

Comment:
Could you add comments that static map has to be used for ipv6 host ips , because kubernetes currently does not add it to host resources yet.